views/check-your-answers.html

```diff
@@ -48,6 +48,15 @@ <h2 class="govuk-heading-m">
             },
             value: {
               text: objection.created_by.fullName
+            },
+            actions: {
+              items: [
+                {
+                  href: "/objecting-entity-name/?id=1",
```

Comment:
It may be better to work backwards here, is adding a query param to the url the best way to achieve what we want to?
If this param is to be used as a flag to notify the page that clicking continue takes you straight to check-your-answers instead of the normal flow, then a user could enter this url manually and bypass the normal flow completely.
Would recommend we organise a story breakdown and decide how we want to handle this before changing this page to incorporate the links

Reply:
Will remove the queries - these are really just placeholders to be dealt with as we progress this further - adding change link was an easy thing to get started with while waiting for the story breakdown.